chatgse/app/locales/cn.ts

```diff
@@ -342,7 +342,7 @@ const cn = {
   },
   Store: {
     DefaultTopic: "新的聊天",
-    BotHello: "有什么可以帮你的吗",
+    BotHello: "Hello! I am the model's assistant. For more explanation, please see the About text in the sidebar. We will now be going through some initial setup steps together. To get started, could you please tell me your name?",
```

Comment:
I would revert to the defaults and include this general introduction in a "home page" of sorts, which is shown in the beginning and can be accessed via a help button afterwards. The home page could include the info section, about section, and the note about the paper in the original ChatGSE.

Reply:
Reverted here and in the English locale file